cmake/package_search/Findlibdpdk.cmake

```diff
@@ -12,89 +12,84 @@
 # the License.
 # =============================================================================
 
-# Often this is installed in /opt/mellanox/libdpdk. If so, run `set(libdpdk_ROOT "/opt/mellanox/libdpdk")` before calling this
+# Often this is installed in /opt/mellanox/dpdk. If so, run `set(libdpdk_ROOT "/opt/mellanox/dpdk")` before calling this
 # file
 
-# List of required args. Start with INCLUDE_DIR since the first one is displayed
-
-list(APPEND libdpdk_REQUIRED_VARS libdpdk_INCLUDE_DIR)
-
 # Find BSD
 find_package(bsd QUIET)
```

Comment:
Remark: This should probably not be quiet, since we'll fail (?) later if we don't find it.

Reply:
I'm not sure what the correct functionality is here. I've seen guides which do both. I think I side more with you that its better to print the output incase BSD is not found it will say why in the logs.

Reply:
Do you think we should use `REQUIRED` here or just remove `QUIET`?